pkgs/tools/typesetting/asciidoctor/Gemfile

```diff
@@ -1,9 +1,7 @@
 source 'https://rubygems.org'
 gem 'asciidoctor'
 gem 'asciidoctor-diagram'
-gem 'asciidoctor-bespoke'
```

Comment:
We generally don't like to remove functionality from release branches. Is there a way to bring this back?

Reply:
I created one commit on top, that brings back bespoke. @basvandijk PTAL.